package.json

```diff
@@ -80,7 +80,7 @@
     "webpack-dev-server": "^3.8.0"
   },
   "engines": {
-    "node": "^10.16.3"
+    "node": "^10.6.0"
```

Comment:
I can tell this is working by CD'ing out of the EDS and back into it, and `avn` sets the node version to 10.6 as expected.